MapboxNavigationUI/RouteVoiceController.swift

```diff
@@ -242,16 +242,21 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         return text
     }
     
-    func localizeRoadDescription(_ step: RouteStep) -> String {
+    func localizeRoadDescription(_ step: RouteStep, markUpWithSSML: Bool) -> String {
         var road = ""
+        let escapeIfNecessary = {(distance: String) -> String in
+            return markUpWithSSML ? distance.addingXMLEscapes : distance
+        }
         if let name = step.names?.first {
             if let code = step.codes?.first {
-                road = String.localizedStringWithFormat(NSLocalizedString("NAME_AND_REF", value: "%@ (%@)", comment: "Format for speech string; 1 = way name; 2 = way route number"), name, code)
+                let markedUpName = markUpWithSSML ? "<say-as interpret-as=\"address\">\(name.addingXMLEscapes)</say-as>" : name
```

Comment:
You’ve probably noticed some parallel between this line and `escapeIfNecessary` above. 😉